javascript/javascript1/week4/task_1.js

```diff
@@ -0,0 +1,86 @@
+let myName = "";
+let todo = [];
+
+function getReply(command) {
+  const array = command.split(" ");
+  let response="";
+  if (command.includes("hello my name is")) {
+    myName = array[4];
+    response = `nice to see you ${myName}`;
+  } else if (command.includes("what is my name")) {
+    response = `my name is ${myName}`;
+  } else if (command.includes("add fishing")) {
```

Comment:
You are adding the task fx. `fishing` in a static way. You could rather compare with only `add ` this text and add the task whatever comes after `add `, right?